packages/adapter-next/public/CallToAction/javascript.jsx

```diff
@@ -41,9 +41,7 @@ const PascalNameToReplace = ({ slice }) => {
 					<PrismicNextLink
 						className="es-call-to-action__button"
 						field={slice.primary.buttonLink}
-					>
-						{slice.primary.buttonLabel || "Learn more…"}
-					</PrismicNextLink>
+					/>
```

Comment:
❓ #question: We lost the fallback values ("Learn more…"). Is that intentional? I think that's okay for a slice template, just want to make sure it's a conscious decision.

Reply:
I did the same choice in our examples, really for the second reason:

> with them, we lose the ability to show how "powerful" the new link is

It's so cool to let them see that you don't need to do anything right now

Reply:
Works for me! 🙂 We can always explain how to do a fallback in the documentation.